### ISSUE TITLE
Discourse: Add categories and tags to `credResult.json`

### ISSUE THREAD

**Issue by s-ben** [sourcecred/sourcecred#2740]
Maker is looking to do some analysis on current Cred flows according to categories and tags. This will presumably be useful for us and other communities as well. A post's categories and tags are stored in the database, but it doesn't appear these properties have made it to `credResult.json`, which is what we parse for analysis purposes in Observable notebooks and in the python scripts I've developed. 

I'm assuming that adding a post's categories and tags to `credResult.json` is the most straightforward way to do this, but know there's refactoring going on and other issues I'm surely unaware of.

**Comment by blueridger** [comment]
This is kind of undesireable within the current setup. After the refactors, when we have solidified the idea of "beliefs" and implemented the graph tagging system, then things like this should be more doable.

**Comment by blueridger** [comment]
One heavy-duty workaround would be to cross reference the object ids in the credResult node addresses with the objects retrieved from the sourcecred load command or directly from the discourse API

**Comment by s-ben** [comment]
@blueridger, yeah, was suspecting this was bad timing with the refactors. May take a pass at fetching from the database somehow in python. Or just hitting the Discourse API myself. Thanks for the heads up and ideas.
